src/interfaces/iDeviceManager.ts

```diff
@@ -139,4 +139,20 @@ export default interface IDeviceManager {
    * @memberof IDeviceManager
    */
   getState(): Promise<any>;
+
+  /**
+   * Reads out the power consumtion on the camera.
+   *
+   * @returns {Promise<any>} An object containing different power consumtion parameters.
+   * @memberof IDeviceManager
+   */
+  getPowerUsage(): Promise<any>;
+
+  /**
+   * Reads out the internal and external temperature on the camera.
```

Comment:
Technically correct, but for IQ it reads out temperature of every temp sensor.

node cli.js temp
Value { mvx_css: 40.970947265625,
  mvx_dss: 39.81175231933594,
  mvx_mss: 39.34638977050781,
  mvx_upa: 40.50798797607422,
  sensor_tmp103a: 37,
  sensor_tmpimx477: 38 }

Reply:
But on the Huddly GO we only expose internal and external. I think that should be enough for everyone...

Reply:
Perhaps with the addition of sensor temperature. The 4 different Movidius temperatures are less interesting.

Reply:
Reads out the measurements of all temperature sensors in the camera

Reply:
And for Huddly GO we give min/max values. Guess the explanations is good enough as is.

Reply:
@geir-t if you agree with the Js-Doc explanation, could you approve the PR, otherwise i cannot merge it into master.